app/src/main/java/com/rage/clamber/Fragments/LoginActivity/NewUserDialogFragment.java

```diff
@@ -83,8 +90,28 @@ public void onClick(DialogInterface dialog, int which) {
         return dialog;
     }
 
-    public boolean isEditTextEmpty(EditText editText){
+    protected boolean isEditTextEmpty(EditText editText) {
         return editText.getText().toString().trim().length() == 0;
     }
+    /**
+     * Method to determine whether the user input skill level is valid.
+     * @param skillLevel - the user intput string
+     * @return - an integer value that will be stored in the database. If the string is not valid
+     * it returns a defined invalid int to check for.
+     */
+    protected int getSkillLevel(String skillLevel) {
```

Comment:
Could this be made into a `public static` method somewhere? That way you could call it from both places that use it without having to repeat the code.